sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateFormatterSuite.scala

```diff
@@ -199,4 +197,8 @@ class DateFormatterSuite extends SparkFunSuite with SQLHelper {
     // SparkUpgradeException here.
     intercept[SparkUpgradeException](formatter.parse("02-29"))
   }
+
+  override def checkFormatterCreation(pattern: String, isParsing: Boolean): Unit = {
```

Comment:
can we move it to the beginning of this class? to avoid code conflict when we add new tests